theseus_gui/src/components/RowDisplay.vue

```diff
@@ -0,0 +1,135 @@
+<script setup>
+import { ChevronLeftIcon, ChevronRightIcon } from 'omorphia'
+import Instance from '@/components/ui/Instance.vue'
+import News from '@/components/ui/News.vue'
+
+const props = defineProps({
+  instances: Array,
+  news: Array,
+  label: String,
+})
+
+const shouldRenderNormalInstances =
+  props.instances && props.instances?.length !== 0 && props.instances?.some((i) => !i.trending)
+const shouldRenderNews = props.news && props.news?.length !== 0
+const shouldRenderTrending =
+  props.instances && props.instances?.length !== 0 && props.instances?.every((i) => i.trending)
+
+const handleLeftPage = () => {
+  console.log('page left')
+}
+
+const handleRightPage = () => {
+  console.log('page right')
+}
+</script>
+
+<template>
+  <div class="row">
+    <div class="header">
+      <p>{{ props.label }}</p>
+      <hr aria-hidden="true" />
+      <div class="pagination">
+        <ChevronLeftIcon @click="handleLeftPage" />
+        <ChevronRightIcon @click="handleRightPage" />
+      </div>
+    </div>
+    <section class="mods" v-if="shouldRenderNormalInstances">
+      <Instance
+        v-for="instance in props.instances"
+        :key="instance.id"
+        display="gallery"
+        :instance="instance"
+      />
+    </section>
+    <section class="news" v-else-if="shouldRenderNews">
+      <News v-for="news in props.news" :key="news.id" :news="news" />
+    </section>
+    <section class="trending" v-else-if="shouldRenderTrending">
+      <Instance
+        v-for="instance in props.instances"
+        :key="instance.id"
+        display="card"
+        :instance="instance"
+      />
+    </section>
```

Comment:
The CSS + this part is very wonky and doesn't work well on other screen sizes. It should make the line be the longest no matter what, with the arrows taking up as much space as they need on the right side and the text taking as much space it needs on the left side.

Reply:
The Arrows should also be optional- you won't have to scroll for the "jump back in section" (there will only be 4 elements). Also the scrolling doesn't work